packages/webpack-plugin/src/webpack-entities.ts

```diff
@@ -17,7 +17,9 @@ import type {
     StringSortableSet,
     StylableBuildMeta,
 } from './types';
-import { stylesheet, injectStyles } from './runtime';
+import { stylesheet } from '@stylable/runtime/dist/runtime';
```

Comment:
can we avoid using internal output paths?

Reply:
Not if we don't expose new package for the stylesheet itself